launcher/src/components/UI/base-header/components/modals/NotifModal.vue

```diff
@@ -165,7 +165,7 @@ export default {
       let test = [];
       const selectedSetup = this.selectedSetup;
       if (selectedSetup && selectedSetup.services) {
-        const selectedServiceIds = selectedSetup.services.map((service) => service.id);
+        const selectedServiceIds = selectedSetup.services.map((service) => service.config.serviceID);
```

Comment:
is it correct? we do NOT have service.config.serviceID in the selectedSetup.

Reply:
yes we do, because i changed it. the services in selectedSetup is now the same data as in InstalledServices.

before we had to manage 2 different service states which not only broke stuff, but also made everything very hard.